ibis/backends/tests/test_param.py

```diff
@@ -116,21 +116,21 @@ def test_scalar_param_map(con):
             "timestamp",
             "timestamp_col",
             id="string_timestamp",
-            marks=pytest.mark.notimpl(["druid"]),
+            marks=pytest.mark.notimpl(["druid", "bigquery"]),
```

Comment:
Are these regressions since they were passing before? Why are they now failing?

Reply:
The time without timezone would be converted into `DATETIME` instead of `TIMESTAMP` after this change. Hence, the new behavior is breaking the tests. I added the error reasons for more details.